style/textsize.less

```diff
@@ -8,11 +8,61 @@
 		box-sizing: border-box;
 		padding: 3px 0;
 		text-align: center;
+		font-size: 17px;
```

Comment:
probably you already notice the header text doesn't look consistent between device and browser, here's the ticket that we are going to fix, but before going into details, a quick fix is updating the padding to `4px 0` and remove `height`

Reply:
Hmm yea I believe Sudhanshu and I talked about it at some point. Thanks for sharing ticket link though, hadn't seen it yet. I'll tweak padding and remove height though, sounds good